go/mysql/collations/mysqlversion.go

```diff
@@ -145,7 +152,7 @@ var globalVersionInfo = map[ID]struct {
 	80:   {alias: []collalias{{0b01111111, "cp850_bin", "cp850"}}, isdefault: 0b00000000},
 	81:   {alias: []collalias{{0b01111111, "cp852_bin", "cp852"}}, isdefault: 0b00000000},
 	82:   {alias: []collalias{{0b01111111, "swe7_bin", "swe7"}}, isdefault: 0b00000000},
-	83:   {alias: []collalias{{0b01111111, "utf8_bin", "utf8"}, {0b01111111, "utf8mb3_bin", "utf8mb3"}}, isdefault: 0b00000000},
+	83:   {alias: []collalias{{0b01111111, "utf8_bin", "utf8"}, {0b01111111, "utf8mb3_bin", "utf8mb3"}, {0b01111111, "utf8mb4_bin", "utf8mb4"}}, isdefault: 0b00000000},
```

Comment:
I am not sure about this change.

Reply:
This is a generated file based on the actual MySQL collations. So this change is wrong.